irods_consortium_continuous_integration_test_hook.py

```diff
@@ -77,7 +77,9 @@ def main():
     built_packages_root_directory = options.built_packages_root_directory
     package_suffix = irods_python_ci_utilities.get_package_suffix()
     os_specific_directory = irods_python_ci_utilities.append_os_specific_directory(built_packages_root_directory)
-    irods_python_ci_utilities.install_os_packages_from_files(glob.glob(os.path.join(os_specific_directory, 'irods-netcdf*.{0}'.format(package_suffix))))
+    package_list = glob.glob(os.path.join(os_specific_directory, 'irods-netcdf*.{0}'.format(package_suffix)))
+    package_list = sorted(package_list, key=lambda s:-os.path.basename(s).find('client'))  # client module must be installed first
```

Comment:
I've never seen that syntax before. What does the hyphen mean?

Reply:
in this case, just a minus. if we left it out, we'd be sorting things matching this function to the end of the list instead of the beginning.  All it does here is make sure the package name containing 'client' as a substring is in the front position.  It is the dependency for the other two packages.

Reply:
Very good. Thanks